docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -1,10 +1,20 @@
 # Integrate with SAP Cloud Logging
 
-SAP Cloud Logging is an instance-based and environment-agnostic observability service that builds upon OpenSearch to store, visualize, and analyze logs, metrics, and traces. This guide explains how to define LogPipelines and TracePipelines to ingest application and access logs as well as distributed trace data in instances of SAP Cloud Logging.
+## Overview
+
+Learn how to define LogPipelines and TracePipelines to ingest application and access logs as well as distributed trace data in instances of [SAP Cloud Logging](https://help.sap.com/docs/SAP_CLOUD_LOGGING?locale=en-US&version=Cloud).
+
+SAP Cloud Logging is an instance-based and environment-agnostic observability service that builds upon OpenSearch to store, visualize, and analyze logs, metrics, and traces.
+
+- Signal type: logs, metrics, and traces
+- Backend type: third-party remote
+- OTLP-native: yes
 
 ## Prerequisites
 
-- An instance of SAP Cloud Logging with OpenTelemetry enabled to ingest distributed traces
+- Kyma as the target deployment environment
+- The [Telemetry module](https://kyma-project.io/#/telemetry-manager/user/README) is [enabled](https://kyma-project.io/#/02-get-started/08-install-uninstall-upgrade-kyma-module?id=install-uninstall-and-upgrade-kyma-with-a-module)
+- An instance of SAP Cloud Logging with OpenTelemetry enabled to ingest distributed traces. For details, see [Create an SAP Cloud Logging Instance through SAP BTP Service Operator](https://help.sap.com/docs/SAP_CLOUD_LOGGING/454331d80e3b42b1804d83a672cf098b/f6aa131faee64f78b9cbba6a5b579b8f.html?locale=en-US&version=Cloud) or, alternatively, [Ship via OpenTelemetry API Endpoint](https://help.sap.com/docs/SAP_CLOUD_LOGGING/454331d80e3b42b1804d83a672cf098b/fdc78af7c69246bc87315d90a061b321.html?locale=en-US&version=Cloud).
```

Comment:
As dicussed, that does not fit yet
- You need to have an instance created in some way, the recommended way is the BTP operator
- The instance must have the otlp_ingest attribute enabled
- The main contract is the resulting secret (which could be also created manually using the alternative methods) having a fixed name and namesapce